src/Shared/BundleConstantsRule.php

```diff
@@ -57,4 +59,18 @@ public function apply(AbstractNode $node)
         }
     }
 
+    /**
+     * @param mixed $constantValue
+     *
+     * @return mixed
+     */
+    private function trimBundleNamePrefix($constantValue)
+    {
+        if (!is_string($constantValue)) {
+            return $constantValue;
+        }
+
+        return preg_replace('/[A-Z0-9_]+:/', '', $constantValue);
```

Comment:
Can we also make it clear in the violation message that it's allowed to have this as well?